hunts/src/TagCell.tsx

```diff
@@ -21,32 +22,39 @@ function TagCell({ row }: { row: Row<Puzzle> }) {
     : row.original.tags.filter((t) => !t.is_meta && t.name !== row.original.name);
 
   return (
-    <>
+    <div
+     onMouseEnter={() => {
+       setUiHovered(true);
+     }}
+     onMouseLeave={() => {
+       setUiHovered(false);
+     }}
+     onClick={() => {
+       dispatch(
+        showModal({
+          type: "EDIT_TAGS",
+          props: {
+            huntId,
+            puzzleId: row.values.id,
+            puzzleName: row.values.name,
+          },
+        })
+       );
+     }}
+     style={{
```

Comment:
Should we drop a `TODO` here like you did above to remember to refactor these styles into a shared CSS class?